packages/graphql-transformers-e2e-tests/package.json

```diff
@@ -67,6 +67,7 @@
     },
     "testEnvironment": "node",
     "collectCoverage": true,
+    "coverageProvider": "v8",
```

Comment:
Since our current jest configuration is set to 'collectCoverage' to 'true', I'm changing the coverage provider to 'v8'. However we do not enforce any threshold for this package.